src/test/util/setup_common.cpp

```diff
@@ -187,6 +187,16 @@ TestingSetup::TestingSetup(const std::string& chainName, const std::vector<const
         throw std::runtime_error("LoadGenesisBlock failed.");
```

Comment:
please rebase this on master, otherwise it is needlessly hard to review. I suspect this doesn't even compile?

Reply:
Not sure I follow, but it's been rebased.

Reply:
To see if a commit compiles locally, you can use `make`. For example this commit indeed fails:

```
$ git checkout 1c6f4e6fe07bb111dd40e6ebfb8936f49a51c2e3 && make 
HEAD is now at 1c6f4e6fe0 test: call VerifyLoadedChainstate during ChainTestingSetup
Making all in src
make[1]: Entering directory '/home/marco/workspace/btc_bitcoin_core/src'
make[2]: Entering directory '/home/marco/workspace/btc_bitcoin_core/src'
make[3]: Entering directory '/home/marco/workspace/btc_bitcoin_core'
make[3]: Leaving directory '/home/marco/workspace/btc_bitcoin_core'
  CXX      test/util/libtest_util_a-setup_common.o
test/util/setup_common.cpp:192:9: error: use of undeclared identifier 'fReindex'
        fReindex.load(),
        ^
1 error generated.
make[2]: *** [Makefile:11221: test/util/libtest_util_a-setup_common.o] Error 1
make[2]: Leaving directory '/home/marco/workspace/btc_bitcoin_core/src'
make[1]: *** [Makefile:17227: all-recursive] Error 1
make[1]: Leaving directory '/home/marco/workspace/btc_bitcoin_core/src'
make: *** [Makefile:817: all-recursive] Error 1